internal/tests/run/scenario_test.go

```diff
@@ -70,10 +74,11 @@ func setup() (func(*testing.T, string), func(string) ([]byte, error)) {
 }
 
 func TestScenarios(t *testing.T) {
```

Comment:
We're missing the following features:
- config flag `-max-errors N` to stop the test runner after N total observed errors (N is an unsigned integer, 0 means never)
- config flag `-max-error-freq N` to stop the test runner when the rate of errors observed per second is greater than N (inclusive); N should be a float; examples: 5 means 5 errors per second, 0.1 means one error every 10 seconds.